core/modules/checkout/components/Payment.ts

```diff
@@ -233,6 +233,9 @@ export const Payment = {
 
       // Let anyone listening know that we've changed payment method, usually a payment extension.
       this.$bus.$emit('checkout-payment-method-changed', this.payment.paymentMethod)
+    },
+    changeCountry () {
+      this.$bus.$emit('checkout-before-paymentMethods')
```

Comment:
```suggestion
      this.$store.dispatch('cart/syncPaymentMethods', { forceServerSync: true })
```

Reply:
Ok thanks, I forgot about the move away from event bus for things like this. Updated.